init/ocp4.inc.sh

```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
+# Directory Manager Password
+IPA_DM_PASSWORD="${IPA_DM_PASSWORD:-${PASSWORD}}"
+
+# Freeipa admin passowrd
+IPA_ADMIN_PASSWORD="${IPA_ADMIN_PASSWORD:-${PASSWORD}}"
+
+# TODO This could have changes when implementing
+#      the replicas for freeipa-operator repository.
+#
+# This need to get a OTP by:
+#
+# ```sh
+# kinit admin
+# ipa host-add replica.example.com --random
+# ```
+#
+# Reference: https://www.adelton.com/docs/idm/replicate-your-identity-management
+IPA_ENROLLMENT_OTP="${IPA_ENROLLMENT_OTP:-${PASSWORD}}"
```

Comment:
The handling of password vs. OTP is not entirely correct. I suggest that you remove the OTP code for now until we can test this.

Reply:
Thanks mate! updated